content/en/user-manual/optimization/guidelines.md

```diff
@@ -10,21 +10,25 @@ Here are some tips and hints on how to achieve good performance in your PlayCanv
 
 * Calling 'new' to allocate a JavaScript object (particularly vectors, matrices and quaternions) represents a dynamic allocation and can be expensive. Therefore you should, where possible, preallocate objects in a script's initialize function and reuse them in the update function. It also leads to Garbage Collection which can cause periodical freezes.
 
-## Graphics
+## Graphics - CPU
```

Comment:
Please keep new lines around paragraph/headings/etc. I wish there was a way to auto-enforce this (because it keeps our Transifex project nice and tidy).

Reply:
done